src/tasks/AppleAppBuilder/Templates/runtime-coreclr.m

```diff
@@ -0,0 +1,161 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#import <Foundation/Foundation.h>
+#include "coreclrhost.h"
+#include <TargetConditionals.h>
+#import <os/log.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <dlfcn.h>
+
+#import "util.h"
+
+#define APPLE_RUNTIME_IDENTIFIER "iossimulator-arm64"
```

Comment:
this should use the template string like runtime.m

Reply:
Good catch. I assume we can handle that as part of the follow-ups tracked in https://github.com/dotnet/runtime/issues/111745.